index.html

```diff
@@ -79,8 +79,7 @@ <h1 id="title"><i class="todo">PROPOSED</i> Decentralized Identifier Working Gro
 The <strong>mission</strong> of the
 <a href="https://www.w3.org/2019/did-wg/">Decentralized Identifier Working Group</a>
 is to maintain the <a href="https://www.w3.org/TR/did-core/">Decentralized Identifiers (DIDs)</a>
-specification and related Working Group Notes. The WG will also seek consensus
-around the specification of DID Methods and DID Resolution.
+specification and related Working Group Notes. The WG will also seek consensus around the best way to achieve effective interoperability.
```

Comment:
I prefer the note about did resolution and dereferencing, see my comment on https://github.com/w3c/did-wg-charter/pull/27#discussion_r1089526511

I don't think defining methods is required to achieve interoperability.